ci/cpu/build.sh

```diff
@@ -41,6 +41,9 @@ env
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
+gpuci_conda_retry remove -y --force arrow-cpp pyarrow 'arrow-cpp-proc=*=cuda'
```

Comment:
Is this line still needed given the one below?

Reply:
Conda was giving inconsistent env error when only the below line is used. This is just a temporary change that I'll remove.

Reply:
Ok. Is that still happening? If so, is there a log we could look at?

Reply:
Itseems that error had to do something I worded the command, now it's gone away.